### ISSUE TITLE
导出的模型有问题，也不能再你的onnx->rknn模型正确进行运行

### ISSUE THREAD

**Issue by Lan1tan** [cqu20160901/yolov8n_official_onnx_tensorRT_rknn_horizon#3]
在修改激活函数和
    y = torch.cat((self.dfl(box), cls.sigmoid()), 1)       # 导出本示例中的onnx修改行
后，模型不能正确生产box，并且导出的onnx模型并不能在您提供的代码中正确转换

**Comment by cqu20160901** [comment]
导出onnx模型用工具看一下，形式和我示例中给的是一样的不。查看onnx用工具netron

**Comment by Lan1tan** [comment]
> 导出onnx模型用工具看一下，形式和我示例中给的是一样的不。查看onnx用工具netron

看了，具体的模型结构在最后是不太一样的，方便问一下到底如何才能转换成功吗，或者可以给出您转换的代码，我时根据您的博客改了激活函数和y = torch.cat((self.dfl(box), cls.sigmoid()), 1) 做的，一步步做下来但是效果却不相同

**Comment by cqu20160901** [comment]
就改一行理论上不应该呀，确认改对地方了

**Comment by Lan1tan** [comment]
![1698388691976](https://github.com/cqu20160901/yolov8n_official_onnx_tensorRT_rknn_horizon/assets/95198770/c95f163a-6921-4cea-88e6-f93cb827a072

在这里修改后，box不能圈出物体，而且跟您的模型不一样

**Comment by Lan1tan** [comment]
您能提交一版您的模型转换的代码吗，感觉好多人也是在讨论这个模型转换和您的不一致
